GOVERNANCE.md

```diff
@@ -0,0 +1,176 @@
+ 
+### Team members
+
+Team member status may be given to those who have made ongoing contributions to the Metrictank project for at least 3 months. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account.
+
+New members may be proposed by any existing member by posting on the private [team mailing list][team]. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority](#supermajority-vote) vote.
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to metrictank-team@googlegroups.com for record-keeping purposes.
+
+If they choose to accept, the onboarding procedure is followed.
+
+Team members may retire at any time by posting a message on the private [team mailing list][team].
+
+Team members can be removed by [supermajority](#supermajority-vote) vote on the private [team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum. Any removal vote can cover only one single person.
+
+Upon the death of a member, they leave the team automatically.
+
+In case a member leaves, the offboarding procedure is applied.
+
+The current team members are:
+
+* Anthony Woods ([Grafana Labs](https://grafana.com)) (awoods@grafana.com)
+* Dieter Plaetinck ([Grafana Labs](https://grafana.com)) (dieter@grafana.com)
+* Florian Boucault ([Grafana Labs](https://grafana.com)) (florian.boucault@grafana.com)
+* Mauro Stettler ([Grafana Labs](https://grafana.com)) (mauro.stettler@grafana.com)
+* Robert Milan ([Grafana Labs](https://grafana.com)) (robert@grafana.com)
+
+### Technical decisions
+
+Technical decisions should be discussed and voted upon on the Metrictank [development mailing list][devlist]. Informal conversations may also be held in *#metrictank-dev* in [slack][slack] or in the appropriate Github issue.
+
+Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority](#majority-vote) vote.
+
+### Governance changes
+
+Material changes to this document are discussed publicly in the *#metrictank-dev* channel in [slack][slack] or on the Metrictank [development mailing list][devlist]. Any change requires a [supermajority](#supermajority-vote) in favor.
+
+### Editorial changes
+
+Editorial changes are changes which fix spelling or grammar, update work affiliation, or similar; they update style or reflect an outside and obvious reality. They do not change the intention or meaning of anything in this document. They must be made via PR and accepted via [lazy consensus](#consensus).
+
```

Comment:
i note that this means we'll need to leave most editorial PR's open for at least 21 days

Reply:
Unless consensus is reached before that time period

Reply:
Yes but that's kindof ridiculous, any time someone files a PR for an editorial tweak, it's silly that either we need input from many team members, or wait 21 days

Reply:
The change to 5 days looks good.